vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -361,8 +492,9 @@ def __init__(
         self.topk_group = topk_group
         self.custom_routing_function = custom_routing_function
         if is_hpu:
-            from vllm_hpu_extension.ops import DynamicFusedMOE
-            self.hpu_fused_moe = DynamicFusedMOE(self.num_experts)
+            if not VLLM_REQUANT_FP8_INC:
```

Comment:
Better to merge the if statements.

Reply:
I have refined it, please take another look. Thanks!